Stripe/STPPaymentMethodsInternalViewController.m

```diff
@@ -21,6 +21,8 @@
 #import "STPPaymentMethodTuple.h"
 #import "STPPromise.h"
 #import "STPSourceProtocol.h"
+#import "STPSource.h"
```

Comment:
super nit: I would have expected `STPSource.h` to come before `STPSourceProtocol.h`